src/isar/models/mission/mission.py

```diff
@@ -22,7 +23,7 @@ class Task:
     steps: List[STEPS]
     status: TaskStatus = field(default=TaskStatus.NotStarted, init=False)
     tag_id: Optional[str] = field(default=None)
-    id: UUID = field(default_factory=uuid4, init=False)
+    id: Union[UUID, str] = field(default_factory=uuid4, init=True)
```

Comment:
```suggestion
    id: str = field(default_factory=uuid4, init=True)
```
Should we change to only use str? I think Union makes for confusion and I dont see the point. Imo we just change the UUID to a str as soon as it is created, then keep it str.

Reply:
Agree